errors.go

```diff
@@ -127,35 +119,46 @@ const (
 )
 
 func classifyError(err error) errorClass {
-	if x, ok := err.(awserr.Error); ok {
-		// Short-circuit if the HTTP status code indicates retryability.
-		if f, ok := err.(awserr.RequestFailure); ok {
-			status := f.StatusCode()
-			if status == 429 {
-				return throttlingClass
-			} else if status == 502 || status == 503 || status == 504 {
-				return temporaryClass
-			}
+
```

Comment:
nit: extra newline here not needed between start of function and first line of code